db/src/Pos/DB/Block/Epoch.hs

```diff
@@ -47,6 +47,7 @@ import           Data.Binary (decode, encode)
 import qualified Data.ByteString.Char8 as BS
 import qualified Data.ByteString.Lazy as LBS
 import           Data.Either (partitionEithers)
+import           Data.List.Extra (chunksOf)
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/src/Pos/DB/Block/Epoch.hs#L50